### PR TITLE
Correct Evohome integration docs

### PR DESCRIPTION
**Pull request by zxdavb**
## Proposed change
Remove some (residual) false information, Correct some typos. Expand/clarify some concepts.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/evohome.markdown [hunk 1]
```diff
@@ -12,33 +12,20 @@ ha_codeowners:
   - '@zxdavb'
 ---
 
-The `evohome` integration links Home Assistant with all _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW systems, such as:
+The **evohome** integration links Home Assistant with all _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW systems, such as:
```
Comment (frenck) [inline source/_integrations/evohome.markdown]:
```suggestion
The `evohome` integration links Home Assistant with all _non-US_ [Honeywell Total Connect Comfort (TCC)](https://international.mytotalconnectcomfort.com/Account/Login) CH/DHW systems, such as:
```
That should not have been changed, it points to the integration key/domain.

File: source/_integrations/evohome.markdown [hunk 2]
```diff
@@ -12,33 +12,20 @@ ha_codeowners:
...
-
-### Round Thermostat
-
-Such systems usually have only one Round Thermostat, although they can have two. Systems with one such thermostat are merged into a single **Climate** entity. Systems with two thermostats will have a 3rd entity for the TCC locations, much like evohome, above.
+For your system to be compatible with this integration, then you must be able to access it via [https://international.mytotalconnectcomfort.com/](https://international.mytotalconnectcomfort.com/) (note the 'international').
 
 ## Configuration
 
-To set up this integration, add the following to your **configuration.yaml** file:
+To set up this integration add the following to your **configuration.yaml** file:
```
Comment (frenck) [inline source/_integrations/evohome.markdown]:
```suggestion
To set up this integration, add the following to your `configuration.yaml` file:
```

The comma was actually correct here.

File: source/_integrations/evohome.markdown [hunk 3]
```diff
@@ -62,59 +49,141 @@ location_idx:
...
   type: integer
   default: 300
 {% endconfiguration %}
 
-This is an IoT cloud-polling integration, and the recommended `scan_interval` is 180 seconds. Testing has indicated that this is a safe interval that - by itself - shouldn't cause you to be rate-limited by Honeywell.
+This is an IoT cloud-polling integration and the recommended minimum `scan_interval` is 180 seconds. Testing has indicated that this is a safe interval that - by itself - shouldn't cause you to be rate-limited by the vendor. There is little value in shorter intervals, as this integration will automatically force a refresh shortly after any configuration changes.
+
+## Locations and Zones
+
+TCC systems are implemented as a _location_, which consist of 1-12 _zones_ and, optionally, a DHW controller:
```
Comment (frenck) [inline source/_integrations/evohome.markdown:62]:
```suggestion
TCC systems are implemented as a _location_, which consist of 1-12 _zones_ and, optionally, a DHW controller:

```
Added newline before the list starts, Markdown guidelines.

File: source/_integrations/evohome.markdown [hunk 4]
```diff
@@ -62,59 +49,141 @@ location_idx:
...
+ - the system location (e.g. a house) is used for operating modes such as home, away, economy, etc.
```
Comment (frenck) [inline source/_integrations/evohome.markdown]:
```suggestion
 - The system location (e.g., a house) is used for operating modes such as home, away, economy, etc.
```

File: source/_integrations/evohome.markdown [hunk 5]
```diff
@@ -62,59 +49,141 @@ location_idx:
...
+ - heating zones (e.g. rooms) are used for target temperature
```
Comment (frenck) [inline source/_integrations/evohome.markdown]:
```suggestion
 - Heating zones (e.g., rooms) are used for the target temperature.
```

File: source/_integrations/evohome.markdown [hunk 6]
```diff
@@ -62,59 +49,141 @@ location_idx:
...
+
+### Evohome
+
+Each zone is represented as a **Climate** entity which will expose the zone's operating mode, current temperature and setpoint.
+
+The Evohome location (controller) is also represented as a **Climate** entity which will expose the location's operating mode. Locations have neither a current temperature nor a setpoint, but as all **Climate** entities are required by Home Assistant to report a temperature, this is calculated as the average of all the zones.
```
Comment (frenck) [inline source/_integrations/evohome.markdown]:
```suggestion
The Evohome location (controller) is also represented as a **Climate** entity that will expose the location's operating mode. Locations have neither a current temperature nor a setpoint, but as all **Climate** entities are required by Home Assistant to report a temperature, this is calculated as the average of all the zones.
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/evohome.markdown (hunk 2 above)
Comment (zxdavb) [reply source/_integrations/evohome.markdown]: Usually, I put too many in!

**Review by frenck — APPROVED** [review]
Nice! 👍 
Thanks, @zxdavb! 🎉

**Pull request merged**